Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.h

```diff
@@ -134,7 +134,7 @@ class GStreamerMediaEndpoint : public ThreadSafeRefCounted<GStreamerMediaEndpoin
     void prepareDataChannel(GstWebRTCDataChannel*, gboolean isLocal);
     void onDataChannel(GstWebRTCDataChannel*);
 
-    WARN_UNUSED_RETURN GRefPtr<GstElement> requestAuxiliarySender(GstWebRTCDTLSTransport*);
+    GstElement* requestAuxiliarySender();
```

Comment:
Please, do not remove the `WARN_UNUSED_RETURN`, speacilly now that you're returning a floating ref (but still a ref). I preferred the smart ptr, btw, because even if you don't and the warning, you won't leak.